.travis.yml

```diff
@@ -1,65 +1,30 @@
 language: generic
 os: osx
 osx_image: xcode8.1
-before_install:
-- rvm use $(< .ruby-version) --install --binary --fuzzy
 install:
-- gem install bundler
-- bundle install --jobs=3 --retry=3
-cache: bundler
+  - gem install fastlane --no-rdoc --no-ri --no-document --quiet
+  - gem install cocoapods --no-rdoc --no-ri --no-document --quiet
+  - gem install jazzy --no-rdoc --no-ri --no-document --quiet
+  - gem install semantic --no-rdoc --no-ri --no-document --quiet
 env:
-  global:
-  - NSUnbufferedIO=YES
   matrix:
-  - JOB=Test-macOS ACTION=test
+  - JOB=macOS
 matrix:
   include:
-  - env: JOB=Test-iOS DEVICE='iPhone 6s (10.0)' PLATFORM='iOS Simulator' ACTION=test
-  - env: JOB=Build-tvOS DESTINATION='platform=tvOS Simulator,name=Apple TV 1080p' ACTION=build
-  - env: JOB=Build-watchOS DESTINATION='platform=watchOS Simulator,name=Apple Watch - 42mm' ACTION=build
-  - os: linux
-    dist: trusty
-    sudo: required
-    addons:
-      apt:
-        packages:
-        - clang-3.8
-        - libicu-dev
-    env: JOB=Test-Linux COMPILER=clang-3.8
-    before_install:
-    - wget -q -O - https://swift.org/keys/all-keys.asc | gpg --import -
-    install:
-    - wget https://swift.org/builds/swift-3.0-release/ubuntu1404/swift-3.0-RELEASE/swift-3.0-RELEASE-ubuntu14.04.tar.gz
-    - tar xzf swift-3.0-RELEASE-ubuntu14.04.tar.gz
-    - export PATH=swift-3.0-RELEASE-ubuntu14.04/usr/bin:$PATH
-    script:
-    - swift test
-  - env: JOB=Misc
-    script:
-    - bundle exec pod lib lint --quick
-before_script:
-- set -o pipefail
-- if [[ $DEVICE && $PLATFORM ]]; then
-    export DESTINATION_UUID=`instruments -s devices | grep "$DEVICE" | awk -F '[ ]' '{print $4}' | awk -F '[\[]' '{print $2}' | sed 's/.$//'`;
-    export DESTINATION="platform=$PLATFORM,id=$DESTINATION_UUID";
-    open -b com.apple.iphonesimulator --args -CurrentDeviceUDID $DESTINATION_UUID;
-  fi
+    - script: fastlane ios ci
+      env: JOB=iOS
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: JOB=Linux
+      install:
+        - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
+      script:
+        - swift test
```

Comment:
It does not yet support Linux as a platform. I too would like it to be in Fastlane, but for now it's what we've got. The script used in the `install` action is intended specifically for use under CI, and isn't suited for a self-run lane.